examples/chip-tool/commands/tests/TestCommand.cpp

```diff
@@ -64,6 +64,13 @@ CHIP_ERROR TestCommand::Log(const char * message)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR TestCommand::UserPrompt(const char * message)
+{
+    ChipLogDetail(chipTool, "USER_PROMPT: %s", message);
+    WaitForMs(0);
```

Comment:
why do we have this wait? is this for some flush? please add a comment or remove - waiting 0 seems to not make much sense.

Reply:
+1

Reply:
I did a method named `ContinueOnChipMainThread` which is why there are those `WaitForMs(0)` here. We basically want the `NextTest` to be called onto the chip thread